File: js/programmers/level.0/prgm_120883_1.js
function solution(id_pw, db) {
  const [id, pw] = id_pw;
  for (i = 0; i < db.length; i++) {
    if (db[i][0] === id && db[i][1] === pw) return "login";
    if (db[i][0] === id && db[i][1] !== pw) return "wrong pw";
  }
  return "fail";
}

File: python/programmers/level.0/prgm_120883_1.py
def solution(id_pw, db):
    id, pw = id_pw

    for i in range(len(db)):
        if db[i][0] == id and db[i][1] == pw:
            return "login"        
        if db[i][0] == id and db[i][1] != pw:
            return "wrong pw"        

    return "fail"